src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -769,6 +771,252 @@ protected bool IsAlive
             }
         }
 
+        /// <summary>
+        /// Gets the child Elements of the current Element.
+        /// </summary>
+        /// <returns>Child Elements.</returns>
+        public IEnumerable<Element> GetChildElements()
+        {
+            return GetElementChildElements(this.InternalElement);
+            
+        }
+
+        private IEnumerable<Element> GetElementChildElements(Autodesk.Revit.DB.Element element)
```

Comment:
To be consistent, can we call it `GetChildElementsFromElement`?

Reply:
or better yet `GetElementChildren` ?